llvm/tools/llvm-ar/llvm-ar.cpp

```diff
@@ -130,7 +130,7 @@ static void printArHelp(StringRef ToolName) {
          << "USAGE: " + ToolName +
                 " [options] [-]<operation>[modifiers] [relpos] "
                 "[count] <archive> [files]\n"
-         << "       " + ToolName + " -M [<mri-script]\n\n";
+         << "       " + ToolName + " -M [<mri-script>]\n\n";
```

Comment:
This is probably `[-M <mri-script>]` since `-M` is part of the optional part.

Reply:
Ok, changed it now

Reply:
I think `<>` can be removed.